wagtailio/blog/models.py

```diff
@@ -165,7 +168,10 @@ def siblings(self):
         FieldPanel("main_image"),
         FieldPanel("date"),
         FieldPanel("category"),
-        FieldPanel("introduction"),
+        AIFieldPanel(
+            "introduction",
+            prompts=[DefaultPrompt.DESCRIPTION],
```

Comment:
The `DefaultPrompt` usages will likely change once we refactor these features as django-ai-core agents